cmake/Config.cmake

```diff
@@ -83,11 +84,19 @@ if(RENDER_ALLEGRO5)
     set(GWK_RENDER_NAME "Allegro5")
     set(GWK_INPUT_NAME "Allegro5")
     set(GWK_PLATFORM_NAME "Allegro5")
-    find_package(Allegro5 REQUIRED)
-    set(GWK_RENDER_INCLUDES "${ALLEGRO5_INCLUDE_DIRS}")
-    set(GWK_RENDER_LIBRARIES "${ALLEGRO5_LIBRARIES}")
```

Comment:
Why has the Allegro config been removed?

Reply:
Whoops! I copied the allegro cmake stuff to get mine working, didn't realize I removed that!